ci/Jenkinsfile.groovy

```diff
@@ -715,7 +715,7 @@ ansiColor('xterm') {
                                        """
                                 }
                             }
-                            docker.image('docker.h2o.ai/opsh2oai/hub').inside("--init") {
+                            docker.image('harbor.h2o.ai/opsh2oai/hub').inside("--init") {
```

Comment:
Following @abal5 comment above - it would be great to put shared utilities under common namespace `library` or `opsh2oai` or `whateveryoufindnicelysound`

Reply:
`hub` image is moved to `library` repo.